devicedefender/include/aws/iotdevicedefender/DeviceDefender.h

```diff
@@ -0,0 +1,173 @@
+#pragma once
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+#include <aws/iotdevicedefender/Exports.h>
+
+#include <aws/crt/io/EventLoopGroup.h>
+#include <aws/crt/mqtt/MqttClient.h>
+
+#include <aws/iotdevice/device_defender.h>
+
+namespace Aws
+{
+    namespace Crt
+    {
+
+    }
+
+    namespace Iotdevice
+    {
+        class AWS_IOTDEVICEDEFENDER_API DeviceApiHandle final
```

Comment:
Putting `DeviceApiHandle` in `Iotdevice` namespace will be a problem when adding secure tunneling code to this repo.

Secure tunneling code will have something similar to initialize the the aws-c-iot library (`aws_iotdevice_library_init`). 

One option is to move `DeviceApiHandle` to `Iotdevicedefenderv1` namespace. Secure tunneling will define its own `DeviceApiHandle` in ST's namespace.

Too bad there is no common library for both DD and ST.

Reply:
Ok, I'll move the `DeviceApiHandle` class into the `Iotdevicedefenderv1` namespace.

The common library does seem like it would be the right solution.